### PR TITLE
PLAT-99116: [VirtualList and Scroller] new focusable thumb

### PR DESCRIPTION
**Pull request by jeonghee27**
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
New thumb style need to applied for 2021 TV.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
To change the minimum size of thumb for new GUI, add a prop to `ScrollbarBase` .

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
N/A

### Links
[//]: # (Related issues, references)
PLAT-99119

### Comments
N/A

### CODE REVIEW

File: packages/ui/Scrollable/Scrollbar.js [hunk 1]
```diff
@@ -92,6 +90,7 @@ const ScrollbarBase = memo(forwardRef((props, ref) => {
 				scrollThumbPositionRatio = (scrollOrigin / (scrollContentSize - scrollViewSize)),
 				scrollThumbSizeRatio = Math.max(ri.scale(minThumbSize) / trackSize, Math.min(1, thumbSizeRatioBase));
 
+			setCSSVariable(thumbRef.current, '--scroll-thumb-min-size', minThumbSize);
```
Comment (MikyungKim) [inline packages/ui/Scrollable/Scrollbar.js]: Should we do ri.scale here for `minThumbSize`?

Comment (jeonghee27) [reply packages/ui/Scrollable/Scrollbar.js]: I noticed this css variable is not needed. I'm going to remove it .Thanks.

File: packages/ui/Scrollable/Scrollbar.js [hunk 2]
```diff
@@ -164,6 +172,7 @@ ScrollbarBase.propTypes = /** @lends ui/Scrollable.Scrollbar.prototype */ {
 ScrollbarBase.defaultProps = {
 	corner: false,
 	css: componentCss,
+	minThumbSize: 18,
```
Comment (Djspaceg) [inline packages/ui/Scrollable/Scrollbar.js]: Should this `18` be scaled too, or is it scaled elsewhere? What unit does this 18 represent? Pixels?

Comment (jeonghee27) [reply packages/ui/Scrollable/Scrollbar.js]: It scaled on line 91. 
> scrollThumbSizeRatio = Math.max(ri.scale(minThumbSize) / trackSize, Math.min(1, thumbSizeRatioBase));

it represent size in pixels

**Comment by MikyungKim** [comment]
LGTM

**Pull request merged**